app/client/components/delivery/records/deliveryRecords.tsx

```diff
@@ -1,77 +1,591 @@
+        subscriptionCurrency: props.subscriptionCurrency,
+        productName: capatalize(
+          props.routeableStepProps.productType.shortenedFriendlyName ||
+            props.routeableStepProps.productType.friendlyName
+        ),
+        apiProductName:
+          props.routeableStepProps.productType.fulfilmentDateCalculator
+            ?.productFilenamePart,
+        problemType: deliveryProblem,
+        affectedRecords: props.data.results.filter(record =>
+          selectedProblemRecords.includes(record.id)
+        ),
+        deliveryProblemMap: props.data.deliveryProblemMap,
+        isTestUser: props.productDetail.isTestUser,
+        showProblemCredit:
+          !props.productDetail.subscription.cancelledAt &&
+          !(
+            hasExistingDeliveryProblem &&
+            props.routeableStepProps.productType.delivery?.records
+              ?.contactUserOnExistingProblemReport
+          ),
+        repeatDeliveryProblem: hasExistingDeliveryProblem,
+        contactPhoneNumbers: props.data.contactPhoneNumbers,
+        resetDeliveryRecordsPage
+      }}
+    >
+      <WizardStep
+        routeableStepProps={props.routeableStepProps}
+        hideBackButton
+        fullWidth
       >
-        {props.productType.friendlyName}
-      </h2>
-      <RecordsTable
-        data={data.results}
-        deliveryProblemMap={data.deliveryProblemMap}
-        showDeliveryInstructions={
-          props.productType.delivery?.showDeliveryInstructions
-        }
-        resultsPerPage={7}
-      />
-    </PageNavAndContentContainer>
-  </>
-);
-
-export const DeliveryRecords = (props: RouteableStepProps) => (
-  <FlowStartMultipleProductDetailHandler
-    {...props}
-    headingPrefix={"View delivery history"}
-    hideHeading
-    hasLeftNav={{
-      pageTitle: "Delivery history",
-      selectedNavItem: navLinks.subscriptions
-    }}
-    supportRefererSuffix="delivery_records_flow"
-    loadingMessagePrefix="Retrieving details of your"
-    cancelledExplainer={`This ${props.productType.friendlyName} has been cancelled. You cannot view any of its delivery history.
+        <PageHeaderContainer selectedNavItem={navLinks.subscriptions}>
+          <h1>Delivery history</h1>
+        </PageHeaderContainer>
+        <PageNavAndContentContainer selectedNavItem={navLinks.subscriptions}>
+          <div
+            css={css`
+              margin: ${space[6]}px 0 ${space[12]}px;
+            `}
+          >
+            <ProductDetailsTable
+              productName={capatalize(
+                props.routeableStepProps.productType.shortenedFriendlyName ||
+                  props.routeableStepProps.productType.friendlyName
+              )}
+              subscriptionId={props.productDetail.subscription.subscriptionId}
+              isGift={isGift(props.productDetail.subscription)}
+            />
+          </div>
+          {props.data.results.find(record => !record.problemCaseId) && (
```

Comment:
In order to do the release only to me, as the last thing we change before releasing this, the following change should do it...
```suggestion
          {props.data.results.find(record => !record.problemCaseId) && window && window.identityDetails.userId==="15849095" (
```